### ISSUE TITLE
ascendancies leaks

### ISSUE THREAD

**Issue by lfmnovaes** [marcoaaguiar/poe2-tree#142]
bloodmage - https://imgur.com/a/uiQoR7z
sorc - https://i.imgur.com/zuWIoEm.png and https://i.imgur.com/l5jycYC.png
infernalist - https://imgur.com/a/Tikno9x
gemling - https://i.imgur.com/ck0qMFz.png
warbringer - https://imgur.com/a/upUtbiM

Just got it and saving it here, didn't check

**Comment by shawnkfox** [comment]
Yeah most of these except infernalist are exactly the same ones I had.  warbringer is already deleted on imgur, that is why I downloaded them instead of just posting the links.  Figured some of them would get deleted.
